src/test/java/ecdar/utility/TestRunner.java

```diff
@@ -0,0 +1,23 @@
+package ecdar.utility;
+
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Result;
+import org.junit.runner.notification.Failure;
+
+public class TestRunner {
+    public static void main(String[] args) {
+        // ------------ Test 1: symbol to unicode ------------------
+        Result result1 = JUnitCore.runClasses(StringHelperTestSymbolToUnicode.class);
+        for (Failure failure : result1.getFailures()) {
+            System.out.println(failure.toString());
+        }
+        System.out.println("Result of test StringHelperTestSymbolToUnicode: " + result1.wasSuccessful());
+
+        // ------------ Test 2: unicode to symbol ------------------
+        Result result2 = JUnitCore.runClasses(StringHelperTestUnicodeToSymbol.class);
+        for (Failure failure : result2.getFailures()) {
+            System.out.println(failure.toString());
+        }
+        System.out.println("Result of test StringHelperTestUnicodeToSymbol: " + result2.wasSuccessful());
+    }
+}
```

Comment:
Im pretty sure the IDE/gradle supports running tests.
I really don't think you should have a TestRunner

Reply:
Hmm that does not feel right. We had parameterized tests in java last semester so it should be possible.

Reply:
https://www.tutorialspoint.com/junit/junit_parameterized_test.htm

Reply:
Jeg giver det et forsøg, men kører TestRunner overhovedet hvis man kører `gradlew test`?

Reply:
Fixed!